Sources/PrimerSDK/Classes/Core/Checkout Components/PrimerCheckoutComponents.swift

```diff
@@ -0,0 +1,222 @@
+//
+//  PRTextField.swift
+//  PrimerSDK
+//
+//  Created by Evangelos on 28/1/22.
+//
+
+#if canImport(UIKit)
+
+import UIKit
+
+public class PrimerCheckoutComponents {
+    
+    public static var delegate: PrimerCheckoutComponentsDelegate?
+    
+    public static func listAvailablePaymentMethods(forSession clientToken: String, completion: @escaping ([PaymentMethodConfigType]?, Error?) -> Void) {
```

Comment:
Not sure about the grammar and convention we utilise here so perhaps is going to be a pointless comment.
-----
Calling this function I expect to receive a list/array of "available payment methods" which I suppose is a list of models. What I get as a result is a list of "payment method **types**".
As per the function below being more explicit into the kind of list, I'm receiving, I was wondering where it would be better to have:

```suggestion
    public static func listAvailablePaymentMethodTypes(forSession clientToken: String, completion: @escaping ([PaymentMethodConfigType]?, Error?) -> Void) {
```

Reply:
Done!